ui/components/multichain/multi-srp/import-srp/import-srp.tsx

```diff
@@ -0,0 +1,309 @@
+import React, { useCallback, useMemo, useState } from 'react';
```

Comment:
Note for the entire file: As discussed internally with @montelaidev, this file will be refactored in a follow-up PR to avoid duplicating the logic with `input-srp`.

So I will adds more comments on this new PR once this one will be merged.

See: https://github.com/MetaMask/metamask-extension/pull/30598/files#r1977094784